### PR TITLE
Introducing issue templates

### PR DESCRIPTION
**Pull request by Austinate**
Implements suggestion from #1467 

I started with a small set of templates, just:
1. iOS bug
2. macOS bug
3. Feature request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Feature request
+about:  Would you like to suggest a feature? Please use this template. 
+title: 'Feature request: '
+assignees: ''
```
Comment (vincode-io) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I'm not familiar with Issue templates.  Will this `assignees` entry prompt the user for an assignee?

Comment (Austinate) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Unfortunately no, afaik. It leaves an issue unassigned and we can specify assignees per template if we would like

Comment (paramaggarwal) [reply .github/ISSUE_TEMPLATE/feature_request.md]: If we would like to assign people automatically we can use the CODEOWNERS file of GitHub. 

https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

**Comment by vincode-io** [comment]
@brentsimmons Do we want to give this a try or should we close out this pull request?  If we move forward, we want to tweak the templates a little.  We don't need the "iOS:" prefix in the title anymore and the issue should automatically tag with macOS/iOS.

**Comment by Wevah** [comment]
I’d at least like a reminder somewhere to include problem URLs in the report.

**Pull request closed without merging**